modules/light-clients/08-wasm/types/store_test.go

```diff
@@ -0,0 +1,80 @@
+package types_test
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	prefixstore "cosmossdk.io/store/prefix"
+
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+)
+
+func (suite *TypesTestSuite) TestGetClientID() {
+	clientStore := suite.store
```

Comment:
i thought we were going with the verbose but safer `suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.ctx, defaultWasmClientID), clientMsg)`?

Reply:
ok! makes sense :)